components/tikv_util/src/lib.rs

```diff
@@ -55,6 +57,7 @@ pub mod timer;
 pub mod worker;
 
 static PANIC_WHEN_UNEXPECTED_KEY_OR_DATA: AtomicBool = AtomicBool::new(false);
+const SPACE_PLACEHOLDER_FILE: &str = "space_placeholder_file";
```

Comment:
maybe call this file `reserved_space` is more clear

Reply:
RESERVER_SPACE_FILE ?

Reply:
I'm fine with it.

Reply:
Emmmm. Its previous name was RESERVER_SPACE_FILE and Jinpeng suggested me to replace with space_placeholder